dissect/target/plugin.py

```diff
@@ -930,6 +934,9 @@ def find_plugin_functions(target: Target, patterns: str, compatibility: bool = F
                 if nsmatch or fmatch:
                     plugin_descriptions.append(func)
 
+            if not plugin_descriptions:
+                invalid_funcs.add(pattern)
+
```

Comment:
This will work for classic notation. For tree notation you can use something like this (I add a suggestion for you to apply directly unfortunately :-( )

```
if wildcard or treematch:
    matches = 0
    ...
    matches += 1
    result.append( .... PluginFunction( .....
    ...

if not matches:
    invalid_funcs.add(pattern)
```

Reply:
Thanks for the suggestion, applied and added test in [3f0eebd](https://github.com/fox-it/dissect.target/pull/263/commits/3f0eebdae6f18e0d6969661ac0da20e2970e477d).